doc/start/hardware-recommendations.rst

```diff
@@ -186,39 +188,40 @@ section of the BlueStore Configuration Reference.
 Hard Disk Drives
 ----------------
 
-Consider carefully the cost-per-gigabyte advantage
-of larger disks. We recommend dividing the price of the disk drive by the
-number of gigabytes to arrive at a cost per gigabyte, because larger drives may
-have a significant impact on the cost-per-gigabyte. For example, a 1 terabyte
-hard disk priced at $75.00 has a cost of $0.07 per gigabyte (i.e., $75 / 1024 =
-0.0732). By contrast, a 3 terabyte disk priced at $150.00 has a cost of $0.05
-per gigabyte (i.e., $150 / 3072 = 0.0488). In the foregoing example, using the
-1 terabyte disks would generally increase the cost per gigabyte by
-40%--rendering your cluster substantially less cost efficient.
+Consider carefully the ostensible cost-per-gigabyte advantage
+of larger HDDs, and the concomitant limitations of IOPS per TB.
 
 .. tip:: Hosting multiple OSDs on a single SAS / SATA HDD
-   is **NOT** a good idea.
+   is **NOT** a good idea. In most cases a single OSD should be provisioned
+   on any media other than perhaps SSDs larger than 30 TB.
 
-.. tip:: Hosting an OSD with monitor, manager, or MDS data on a single
-   drive is also **NOT** a good idea.
+.. tip:: Colocating an OSD with Monitor, Manager, or MDS data on the
+   same drive is also **NOT** a good idea.
 
-.. tip:: With spinning disks, the SATA and SAS interface increasingly
+.. tip:: With HDDs, the SATA or SAS interface increasingly
    becomes a bottleneck at larger capacities. See also the `Storage Networking
    Industry Association's Total Cost of Ownership calculator`_.
```

Comment:
I have to say I do not understand this claim. Surely a, say, 12 Gbps SAS is an order of magnitude faster than a rotational disk. Or maybe it discusses the use of SAS expanders and the sort but this is implied and very unclear, to the point that to me this looks like nonsense.

I can't immediately see what does HDD capacity have to do with SAS or SATA interface. HDDs don't generally get much faster as their size increases. Again, maybe it discusses some implied use of expanders on a single bus to share that bus with several disks.

Reply:
And to summarize here: I think this would be a great chance to elaborate about expanders that the text currently glosses over completely. They are indeed an important topic. It (a future PR? I can volunteer to work on it) could discuss especially the oversubscription issue.